README.md

```diff
@@ -28,40 +28,9 @@ You can use one of the following options to try out Ballerina.
 * [Quick Tour](https://ballerina.io/learn/quick-tour/)
 * [Ballerina by Example](https://ballerina.io/learn/by-example/) 
 
-## Download and install
+## Downloading and installing
 
-### Download the binary
-
-You can download the Ballerina distribution at https://ballerina.io.
-
-### Install from source
-
-Alternatively, you can install Ballerina from the source using the following instructions.
-
-#### Prerequisites
-
-* [Oracle JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) or [OpenJDK 8](http://openjdk.java.net/install/)
-* [Node (v8.9.x) + npm (v5.6.0 or later)](https://nodejs.org/en/download/)
-* [Docker](https://www.docker.com/get-docker)
-
-#### Building the source
```

Comment:
Why we are removing installing from the source from Readme file? That information is not there in https://ballerina.io/learn/installing-ballerina/. So from where users can get this details?

Reply:
This PR is to add that info to the "Installing Ballerina" page.

https://github.com/ballerina-platform/ballerina-dev-website/pull/215

Reply:
That information is already there in https://github.com/ballerina-platform/ballerina-dev-website/blob/master/contribution-guide.md with more details. So with this PR are we going to duplicate the same information in both places. IMO, it is not correct. The content in the both places are having different information even right now. If we really want to have it in web site then we need to have a fully descriptive separate page on building from source. (see go for example https://golang.org/doc/install/source)

Reply:
+1 will merge the info and create a separate page.

Reply:
IMHO, we do not have content to add a separate page for the topic "Installing from Source ATM like in Go. Therefore, merged the content that was there in the Contribution Guide to the "Installing from source" section in the "Installing Ballerina" page via ballerina-platform/ballerina-dev-website#215.

Now, both the Contribution Guide in B.io and README file in the ballerina-lang repo links to the "Installing from source" section in the "Installing Ballerina" via ballerina-platform/ballerina-dev-website#215.